src/API/src/export/utils/allDataCsvCreation.ts

```diff
@@ -0,0 +1,24 @@
+const flat = require('flat');
+
+import { Occurrence } from 'src/db/occurrence/entities/occurrence.entity';
+
+// All dates UTC
+
+export function arrayOfFlattenedObjects(array) {
+  const csvArray = [];
+  for (const i in array) {
+    const flatObject = flat.flatten(array[i], {
+      delimiter: '_',
+    });
+    csvArray.push(flatObject);
+  }
+  return csvArray;
+}
+
+export async function flattenOccurrenceRepoObject(
+  occurrenceDbdata?: Occurrence[],
+) {
+  const data = occurrenceDbdata;
+  const dataflat = arrayOfFlattenedObjects(data);
+  return dataflat;
```

Comment:
Let's turn 3 lines into 1 here - `return arrayOfFlattenedObjects(occurrenceDbdata);`

Reply:
Not sure we need this method at all if we're doing that :-)